tests/baselines/reference/docker/vue-next.log

```diff
@@ -63,66 +87,100 @@ Warning: dist/packages/compiler-core/src/options.d.ts:36:25 - (tsdoc-escape-righ
-Warning: dist/packages/compiler-core/src/options.d.ts:87:35 - (tsdoc-malformed-inline-tag) Expecting a TSDoc tag starting with "{@"
-Warning: dist/packages/compiler-core/src/options.d.ts:87:36 - (tsdoc-malformed-inline-tag) Expecting a TSDoc tag starting with "{@"
-Warning: dist/packages/compiler-core/src/options.d.ts:128:39 - (tsdoc-escape-right-brace) The "}" character should be escaped using a backslash to avoid confusion with a TSDoc inline tag
-Warning: dist/packages/compiler-core/src/options.d.ts:128:21 - (tsdoc-malformed-inline-tag) Expecting a TSDoc tag starting with "{@"
-Warning: dist/packages/compiler-core/src/options.d.ts:149:42 - (tsdoc-escape-right-brace) The "}" character should be escaped using a backslash to avoid confusion with a TSDoc inline tag
-Warning: dist/packages/compiler-core/src/options.d.ts:149:43 - (tsdoc-escape-right-brace) The "}" character should be escaped using a backslash to avoid confusion with a TSDoc inline tag
-Warning: dist/packages/compiler-core/src/options.d.ts:149:35 - (tsdoc-malformed-inline-tag) Expecting a TSDoc tag starting with "{@"
-Warning: dist/packages/compiler-core/src/options.d.ts:149:36 - (tsdoc-malformed-inline-tag) Expecting a TSDoc tag starting with "{@"
+Warning: dist/packages/compiler-core/src/options.d.ts:89:42 - (tsdoc-escape-right-brace) The "}" character should be escaped using a backslash to avoid confusion with a TSDoc inline tag
+Warning: dist/packages/compiler-core/src/options.d.ts:89:43 - (tsdoc-escape-right-brace) The "}" character should be escaped using a backslash to avoid confusion with a TSDoc inline tag
+Warning: dist/packages/compiler-core/src/options.d.ts:89:35 - (tsdoc-malformed-inline-tag) Expecting a TSDoc tag starting with "{@"
+Warning: dist/packages/compiler-core/src/options.d.ts:89:36 - (tsdoc-malformed-inline-tag) Expecting a TSDoc tag starting with "{@"
+Warning: dist/packages/compiler-core/src/options.d.ts:130:39 - (tsdoc-escape-right-brace) The "}" character should be escaped using a backslash to avoid confusion with a TSDoc inline tag
+Warning: dist/packages/compiler-core/src/options.d.ts:130:21 - (tsdoc-malformed-inline-tag) Expecting a TSDoc tag starting with "{@"
+Warning: dist/packages/compiler-core/src/options.d.ts:151:42 - (tsdoc-escape-right-brace) The "}" character should be escaped using a backslash to avoid confusion with a TSDoc inline tag
+Warning: dist/packages/compiler-core/src/options.d.ts:151:43 - (tsdoc-escape-right-brace) The "}" character should be escaped using a backslash to avoid confusion with a TSDoc inline tag
+Warning: dist/packages/compiler-core/src/options.d.ts:151:35 - (tsdoc-malformed-inline-tag) Expecting a TSDoc tag starting with "{@"
+Warning: dist/packages/compiler-core/src/options.d.ts:151:36 - (tsdoc-malformed-inline-tag) Expecting a TSDoc tag starting with "{@"
+Warning: /vue-next/packages/reactivity/src/baseHandlers.ts:198:3 - (TS2322) Type '(string | number | symbol)[]' is not assignable to type '(string | symbol)[]'.
```

Comment:
this was fixed in Jan 4. I think the tsc version is wrong here too -- it must be 4.1 or older.